### PR TITLE
Some editorial changes

### PR DESCRIPTION
**Pull request by xfq**
Nice document! These are just some minor editorial changes to make this document more readable.

### CODE REVIEW

**Comment by siusin** [comment]
Thanks!

File: index.html [hunk 1]
```diff
@@ -229,7 +229,7 @@
         <ul>
           <li>The time values returned when calling the <code>now()</code> method on <a>Performance</a> objects with the same <a>time origin</a> MUST be monotonically increasing and not subject to system clock adjustments or system clock skew.</li>
           <li>The difference between any two chronologically recorded time values returned from the <code>now()</code> method MUST never be negative if the two time values have the same <a>time origin</a>.</li>
-          <li><code>translateTime</code> MUST be used to compare two chronologically recorded time values of different <a>time origin</a>.</li>
+          <li><code>translateTime</code> MUST be used to compare two chronologically recorded time values of different <a>time origins</a>.</li>
```
Comment (igrigorik) [inline index.html:232]: Aside: this should be removed, we dropped translateTime.

Comment (xfq) [reply index.html:232]: @igrigorik Thanks for pointing it out. See #5.

Comment (siusin) [reply index.html:232]: Indeed. 

I'll take some time to update this primer in the end of the year.

**Pull request merged**